cliff.toml

```diff
@@ -71,6 +71,7 @@ commit_preprocessors = [
 commit_parsers = [
   { message = "^Bump", skip = true },
```

Comment:
Remove this?

Reply:
I was keeping it in just in case, but it can be removed.